src/components/footer/Footer.js

```diff
@@ -1,8 +1,10 @@
 import './styles.css';
+import React from "react";
 
-export default function Footer () {
+const Footer = () => (
+<div className="footer">
```

Comment:
missing return(). please put jsx part into a return().
const Footer = () => {
   return()
}

Reply:
@AlinaTaoRao This is the shorthand form, which is allowed. But you can, as a team, decide if you want every react component in the same style.

Reply:
Also, if you want a tool that can automate this: https://prettier.io/

Reply:
> @AlinaTaoRao This is the shorthand form, which is allowed. But you can, as a team, decide if you want every react component in the same style.

Btw, the shorthand form is only possible with the arrow function.

Reply:
> Which is the best practice?

There is not really a best practice. Just something you have to decide as a team.